devsetup/scripts/edpm-compute-repos.sh

```diff
@@ -30,14 +30,16 @@ if [[ ! -f $SSH_KEY ]]; then
 fi
 
 cat <<EOF > $CMDS_FILE
-rpm -q git || sudo yum -y install git
-sudo yum -y install python-setuptools python-requests python3-pip
-git clone https://github.com/openstack-k8s-operators/repo-setup
-pushd repo-setup
-sudo pip install -r requirements.txt
-sudo python3 setup.py install
+# Install latest version of repo-setup without installing pip or git
+pushd /tmp
+curl -sL https://github.com/openstack-k8s-operators/repo-setup/archive/refs/heads/main.tar.gz | tar xvz
+pushd repo-setup-main
+python3 -m venv ./venv
+source ./venv/bin/activate
```

Comment:
iirc we can point directly to the pip located in venv/bin/pip, it wouldn't require to source the activate bit.